### ISSUE TITLE
IllegalStateException: Already connected

### ISSUE THREAD

**Issue by echipachenko** [enioka/jqm#300]
Hi!
When I try to access JQM via REST api, when JQM Engine is offline
Follow exception was thrown:
```
Exception in thread "main" com.enioka.jqm.api.JqmClientException: an internal JQM client exception occured
	at com.enioka.jqm.api.JerseyClient.getJobs(JerseyClient.java:680)
	at com.frontier.job.core.service.JobManagerJqmAdapter.main(JobManagerJqmAdapter.java:448)
Caused by: javax.ws.rs.ProcessingException: Already connected
	at org.glassfish.jersey.client.ClientRuntime.invoke(ClientRuntime.java:234)
	at org.glassfish.jersey.client.JerseyInvocation$2.call(JerseyInvocation.java:671)
	at org.glassfish.jersey.internal.Errors.process(Errors.java:315)
	at org.glassfish.jersey.internal.Errors.process(Errors.java:297)
	at org.glassfish.jersey.internal.Errors.process(Errors.java:228)
	at org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:423)
	at org.glassfish.jersey.client.JerseyInvocation.invoke(JerseyInvocation.java:667)
	at org.glassfish.jersey.client.JerseyInvocation$Builder.method(JerseyInvocation.java:423)
	at org.glassfish.jersey.client.JerseyInvocation$Builder.post(JerseyInvocation.java:326)
	at com.enioka.jqm.api.JerseyClient.getJobs(JerseyClient.java:669)
	... 1 more
Caused by: java.lang.IllegalStateException: Already connected
	at sun.net.www.protocol.http.HttpURLConnection.setRequestProperty(HttpURLConnection.java:3071)
	at org.glassfish.jersey.client.HttpUrlConnector.setOutboundHeaders(HttpUrlConnector.java:348)
	at org.glassfish.jersey.client.HttpUrlConnector.access$100(HttpUrlConnector.java:87)
	at org.glassfish.jersey.client.HttpUrlConnector$3.getOutputStream(HttpUrlConnector.java:311)
	at org.glassfish.jersey.message.internal.CommittingOutputStream.commitStream(CommittingOutputStream.java:200)
	at org.glassfish.jersey.message.internal.CommittingOutputStream.commitStream(CommittingOutputStream.java:194)
	at org.glassfish.jersey.message.internal.CommittingOutputStream.commit(CommittingOutputStream.java:262)
	at org.glassfish.jersey.message.internal.OutboundMessageContext.commitStream(OutboundMessageContext.java:812)
	at org.glassfish.jersey.client.ClientRequest.writeEntity(ClientRequest.java:543)
	at org.glassfish.jersey.client.HttpUrlConnector._apply(HttpUrlConnector.java:315)
	at org.glassfish.jersey.client.HttpUrlConnector.apply(HttpUrlConnector.java:227)
	at org.glassfish.jersey.client.ClientRuntime.invoke(ClientRuntime.java:225)
	... 10 more
```

It looks like JQM Client Already Connected, but this is invalid message, because client is NOT connected, and JQM Node is offline.
Could you change this message to something more real ('Could not connect to JQM Node' or something like that). 
It's not major issue, just minor. And it would be great if you fix it.

**Comment by echipachenko** [comment]
Hi @marcanpilami !
It looks like this issue is related to known jersey bug https://github.com/jersey/jersey/issues/3000
Could you update jersey dependencies to the latest version 2.26 ?
Thanks!

**Comment by marcanpilami** [comment]
Yep. Sadly, as we are saddled with Java 1.6, I cannot upgrade Jersey. Which is a concerne for many reasons. We have plans to ditch old Java versions in the next major version. (see #71), and we have already warned that Java 1.6/1.7 usage with JQM is [deprecated ](http://jqm.readthedocs.io/en/jqm-all-2.0.0/release_notes.html#deprecated).

In the meantime, there may be simply the option to override the version of the Jersey client in the dependency - not tested.

**Comment by echipachenko** [comment]
Hi @marcanpilami ! Thanks for explanation.

**Comment by deratzmann** [comment]
Updating to 2.26. seems to not solving this issue. We use payara full application server 5 with a jersey client 2.26. bundled inside. When we try to call a https adress, this problems occurs.

When we downgrade to payara full application server 4 with  a jersey client 2.25.1. bundled inside, the problem is gone.
I guess this bug comes with 2.26. too.

**Comment by marcanpilami** [comment]
Fixed in 596e5c7467caa1069e3e63919ddb33a130bf66fc

As noted above, it was made possible by dropping old Java versions in version 3, so no backport to the 2.x versions is scheduled.
